website/routes.py

```diff
@@ -1151,6 +1151,9 @@ def make_url_map(app):
         Rule([
             '/project/<pid>/draft/<draft_id>/',
         ], 'delete', project_views.drafts.delete_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/submit/',
```

Comment:
Why is it just draft? Probably want to be specific and have draft under the
registration url.

On Fri, Sep 25, 2015 at 9:38 AM, Harry Rybacki <notifications@github.com>
wrote:

> In website/routes.py
> <https://github.com/CenterForOpenScience/osf.io/pull/4338#discussion_r40429758>
> :
>
> > @@ -1151,6 +1151,9 @@ def make_url_map(app):
> >          Rule([
> >              '/project/<pid>/draft/<draft_id>/',
> >          ], 'delete', project_views.drafts.delete_draft_registration, json_renderer),
> > +        Rule([
> > +            '/project/<pid>/draft/<draft_id>/submit/',
>
> Pinging @JeffSpies <https://github.com/JeffSpies> and @brianjgeiger
> <https://github.com/brianjgeiger>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/4338/files#r40429758>
> .
>